### PR TITLE
Fix RollupResponseTranslator javadoc

### PR DESCRIPTION
**Pull request by javanna**
The signature of combineResponses was outdated

### CODE REVIEW

File: x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java [hunk 1]
```diff
@@ -101,7 +101,6 @@ public static SearchResponse translateResponse(MultiSearchResponse.Item[] rolled
             responses.add(item.getResponse());
         }
 
-        assert responses.size() > 0;
```
Comment (ywelsch) [inline x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java:104]: this assertion still holds true if the assertion at the top of the method holds true.

Comment (javanna) [reply x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java:104]: it looks like it's always true, I thought it's not useful, but if you think it is I can restore it.

Comment (ywelsch) [reply x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/RollupResponseTranslator.java:104]: Yes, it's always true, but the point is to document the code like that, i.e. you don't need to think whether from that point on you always have a non-empty list of responses, the code formally tells you it is so.

**Review by ywelsch — APPROVED** [review]
I would reinstate the assertion (personal preference, no blocker), o.w. LGTM.

**Comment by javanna** [comment]
run elasticsearch-ci/part-2

**Pull request merged**